ERROR - An unhandled exception occurred: Operation on an invalid type

## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description

```
------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/progress'
------------------------------------------------------------------------------
✖ Compiling TypeScript sources through NGC
ERROR: Operation on an invalid type
An unhandled exception occurred: Operation on an invalid type
See "/private/var/folders/d_/rh331tx55g1b2y0k9991yblc0000gp/T/ng-ELiKNY/angular-errors.log" for further details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @taiga-ui/components@2.19.0 build: `node --max_old_space_size=8192 node_modules/.bin/nx build --configuration=production "kit" "--skip-nx-cache"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @taiga-ui/components@2.19.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

#### How To Reproduce

Just clone the repository and run the following commands in the following order:

```git clone https://github.com/TinkoffCreditSystems/taiga-ui.git```

```
$ npm ci
$ npm install ng-packagr@11.2.4
```

```bash
npm run build cdk -- --skip-nx-cache
npm run build i18n -- --skip-nx-cache
npm run build core -- --skip-nx-cache
npm run build kit -- --skip-nx-cache
```

```
➜  taiga-ui git:(main) ✗ npm run build kit -- --skip-nx-cache

> @taiga-ui/components@2.19.0 build /Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui
> node --max_old_space_size=8192 node_modules/.bin/nx build --configuration=production "kit" "--skip-nx-cache"


> ng run kit:build:production 
(node:6050) [DEP0128] DeprecationWarning: Invalid 'main' field in '/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@nrwl/nx-cloud/package.json' of 'src/index.js'. Please either fix that or report it to the module author
(Use `node --trace-deprecation ...` to show where the warning was created)
Building Angular Package
WARNING: Option "whitelistedNonPeerDependencies" is deprecated: Use "allowedNonPeerDependencies" instead.

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/classes'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/classes

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/constants'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠙ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/constants

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/enums'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/enums

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/interfaces'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/interfaces

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/providers'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core'
WARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core/tokens' in output.globals – guessing 'tokens'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/providers

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/types'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/types

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/validators'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/validators

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/accordion'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core$1'
WARNING: No name was provided for external module '@tinkoff/ng-polymorpheus' in output.globals – guessing 'ngPolymorpheus'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/accordion

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/arrow'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠙ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core'
WARNING: No name was provided for external module '@tinkoff/ng-polymorpheus' in output.globals – guessing 'ngPolymorpheus'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/arrow

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/utils/format'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/utils/format

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/checkbox'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core$1'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/checkbox

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/select-option'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core$1'
WARNING: No name was provided for external module '@tinkoff/ng-polymorpheus' in output.globals – guessing 'ngPolymorpheus'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/select-option

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/data-list-wrapper'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
WARNING: No name was provided for external module '@taiga-ui/core' in output.globals – guessing 'core$1'
WARNING: No name was provided for external module '@tinkoff/ng-polymorpheus' in output.globals – guessing 'ngPolymorpheus'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/data-list-wrapper

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/utils/files'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/utils/files

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/island'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/components/island

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/utils/dom'
------------------------------------------------------------------------------
✔ Compiling TypeScript sources through NGC
✔ Bundling to FESM2015
⠋ Bundling to UMDWARNING: No name was provided for external module '@taiga-ui/cdk' in output.globals – guessing 'cdk'
✔ Bundling to UMD
✔ Minifying UMD bundle
✔ Writing package metadata
ℹ Built @taiga-ui/kit/utils/dom

------------------------------------------------------------------------------
Building entry point '@taiga-ui/kit/components/progress'
------------------------------------------------------------------------------
✖ Compiling TypeScript sources through NGC
ERROR: Operation on an invalid type
An unhandled exception occurred: Operation on an invalid type
See "/private/var/folders/d_/rh331tx55g1b2y0k9991yblc0000gp/T/ng-ELiKNY/angular-errors.log" for further details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @taiga-ui/components@2.19.0 build: `node --max_old_space_size=8192 node_modules/.bin/nx build --configuration=production "kit" "--skip-nx-cache"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @taiga-ui/components@2.19.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

`/private/var/folders/d_/rh331tx55g1b2y0k9991yblc0000gp/T/ng-ELiKNY/angular-errors.log`

```
[error] Error: Operation on an invalid type
    at extractValue (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/sync-rpc/lib/index.js:165:19)
    at StylesheetProcessor.styleSheetProcessorWorker (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/sync-rpc/lib/index.js:175:12)
    at StylesheetProcessor.process (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/ng-packagr/lib/styles/stylesheet-processor.js:30:40)
    at Object.readResource (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/ng-packagr/lib/ts/cache-compiler-host.js:91:57)
    at TsCompilerAotCompilerTypeCheckHostAdapter.loadResource (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@angular/compiler-cli/src/transformers/compiler_host.js:503:37)
    at Object.get (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@angular/compiler/bundles/compiler.umd.js:28751:94)
    at DirectiveNormalizer._fetch (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@angular/compiler/bundles/compiler.umd.js:20456:47)
    at /Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@angular/compiler/bundles/compiler.umd.js:20586:72
    at Array.map (<anonymous>)
    at DirectiveNormalizer._loadMissingExternalStylesheets (/Users/m.ivanov4/Downloads/Develop/Github/Tinkoff/taiga-ui/node_modules/@angular/compiler/bundles/compiler.umd.js:20586:18)
```

#### Expected Behaviour

It would be great to write in the logs what went wrong in the code. Or a more understandable error.


#### Version Information

```
$ node_modules/.bin/ng-packagr --version
ng-packagr: 11.2.4
@angular/*: 9.1.12
typescript: 3.8.3
rxjs: 6.6.3
node: 16.9.1
npm/yarn: 6.14.15
```

--- a/src/lib/styles/stylesheet-processor.ts
+++ b/src/lib/styles/stylesheet-processor.ts
@@ -158,6 +158,7 @@ export class StylesheetProcessor {
           await import('less')
         ).default.render(css, {
           filename: filePath,
+          math: 'always',
           javascriptEnabled: true,
           paths: this.styleIncludePaths,
         });

